e2e/sample-app/index.js

```diff
@@ -0,0 +1 @@
+console.log('Hello')
```

Comment:
why do you need an index.js at the root of the app ?

Reply:
@moritzraho 
No specific reason, Just added it to check the build output.

Reply:
okie maybe let's remove it then  if the tests pass now :)

Reply:
The test will fail, 
It looks for the build output for this file